File: SortStringsByLastCharacter.py
# Sort Strings By Last Character
# - Given a list of strings `words`, sort them based on the last character of each string.
# - Constraints:
#   - 1 <= len(words) <= 10^4

from typing import List

class Solution:
    def sortByLastChar(self, words: List[str]) -> List[str]:
        return sorted(words, key=lambda x: x[-1])

File: SortStringsByLastCharacter.cs
// Sort Strings By Last Character
// - Given a list of strings, return the list sorted by the last character of each string in ascending order.
// - Time: O(n log n), Space: O(n)

using System;
using System.Linq;

public class Solution
{
    public string[] SortStringsByLastCharacter(string[] words)
    {
        return words.OrderBy(w => w[w.Length - 1]).ToArray();
    }
}
